src/encoded/static/components/regulome_search.js

```diff
@@ -282,6 +283,7 @@ class AdvSearch extends React.Component {
 
                             <input type="submit" value="Search" className="btn btn-sm btn-info" />
                             <input type="hidden" name="genome" value={this.state.genome} />
+                            <input type="hidden" name="maf" value={this.state.maf} />
```

Comment:
I have not see hidden input type in a long time.

Reply:
I just copied whatever Emma did here: https://github.com/ENCODE-DCC/regulome-encoded/blob/dev/src/encoded/static/components/regulome_search.js#L284